animeal/src/Modules/Login/View/Utils/LoginViewOnboardingStepsMapper.swift

```diff
@@ -8,13 +8,14 @@
 import Foundation
 import Style
 
+// sourcery: AutoMockable
 protocol LoginViewOnboardingStepsMappable {
-    func map(_ input: LoginModelOnboardingStep) -> LoginViewOnboardingStep
-    func map(_ input: [LoginModelOnboardingStep]) -> [LoginViewOnboardingStep]
+    func mapStep(_ input: LoginModelOnboardingStep) -> LoginViewOnboardingStep
+    func mapSteps(_ input: [LoginModelOnboardingStep]) -> [LoginViewOnboardingStep]
```

Comment:
You change  `map` to `mapSteps` due to Sourcery rules?

Reply:
Yes, in case of method overloading Sourcery generated mocks with equal property names, and it resulted in compilation error, so we need to avoid using such names in future.